src/pages/stage-page/validate-stage-page.ts

```diff
@@ -25,7 +25,7 @@ export const validateTimetableStage = ({
 
   const startEventValidation = eventSchema.validate(
     {
-      eventDate: startEvent.eventDate,
+      startDate: startEvent.eventDate,
```

Comment:
can we merge these 2 validations? feels like the second one is already validating both start and end date